### PR TITLE
ADD: player model and view, EDIT: controller

### PR DESCRIPTION
**Pull request by branchwelder**
Player can now be moved by WASD and arrow keys. This is very preliminary and a little hacky, I'm expecting there to be some architectural changes.

### CODE REVIEW

File: src/La Trop/controller/controller.cpp [hunk 1]
```diff
@@ -15,16 +15,20 @@ void Controller::handleKeyboard(unsigned char key, int x, int y) {
         case 27:
             exit(0);
         case 'w':
-            std::cout << "up";
+            _model->movePlayer(0,1);
+            _view->render(); // Might want to change render function so it only updates screen/doesnt re-render everything
```
Comment (segerphilip) [inline src/La Trop/controller/controller.cpp]: agreed, not sure how that should be changed yet though

**Review by segerphilip — COMMENTED** [review]
Looks good 🚢 🇮🇹

**Review by segerphilip — APPROVED** [review]

File: src/La Trop/model/player.hpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+//  La Trop
+//
+//  Created by Hannah Twigg-Smith on 4/4/17.
+//  Copyright © 2017 Hedonistic Hibiscus. All rights reserved.
+//
+
+#ifndef player_model_hpp
+#define player_model_hpp
+
+#include "../view/player.hpp"
```
Comment (sammyers) [inline src/La Trop/model/player.hpp]: why

File: src/La Trop/controller/controller.cpp (hunk 1 above)
Comment (sammyers) [reply src/La Trop/controller/controller.cpp]: These changes should get rendered in the main loop.

**Review by sammyers — CHANGES_REQUESTED** [review]
Looks good overall.

**Comment by segerphilip** [comment]
Also required because of my changes, `brew install freeglut`

**Review by sammyers — APPROVED** [review]

**Pull request merged**